crates/services/gas_price_service/src/v1/service.rs

```diff
@@ -105,6 +126,7 @@ where
     pub fn new(
         l2_block_source: L2,
         shared_algo: SharedV1Algorithm,
+        latest_gas_price: Arc<parking_lot::RwLock<(u32, u64)>>,
```

Comment:
It would be nice to have a type that hides this lock(`Arc<parking_lot::RwLock<(u32, u64)>>`) inside. Later we can easier refactor it, plus you will not expose direct access to the lock.